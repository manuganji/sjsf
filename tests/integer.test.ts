import { expect, test } from '@playwright/test';

// integer
test('integer', async ({ page }) => {
	await page.goto('/examples/integer/');
	await page.fill('input[type="integer"]', '123');
	expect(await page.textContent('#value')).toBe('123');
});
