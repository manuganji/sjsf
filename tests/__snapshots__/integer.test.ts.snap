// Vitest Snapshot v1

exports[`integer type > example renders 1`] = `
"<div><div class=\\"columns-2 m-4\\"><div class=\\"rounded box-border border-amber border border-rounded bg-gray-3 py-2 px-4\\"><pre class=\\"inline-block \\"><code>{
  \\"type\\": \\"integer\\",
  \\"title\\": \\"Integer default\\",
  \\"default\\": 0
}</code></pre></div> <form class=\\"flex flex-col min-w-max\\" id=\\"s0\\"><div class=\\"flex flex-col s-c1k-_LTyZdv1\\"><label class=\\"label s-c1k-_LTyZdv1\\" for=\\"root\\">Integer default</label> <input id=\\"root\\" type=\\"number\\" step=\\"1\\">  </div><!--<Layout>--><!--<IntegerField>--></form><!--<Form>--></div><!--<ExampleLayout>--><div class=\\"columns-2 m-4\\"><div class=\\"rounded box-border border-amber border border-rounded bg-gray-3 py-2 px-4\\"><pre class=\\"inline-block \\"><code>{
  \\"type\\": \\"integer\\",
  \\"title\\": \\"Integer min max\\",
  \\"default\\": 1,
  \\"minimum\\": 0,
  \\"maximum\\": 10
}</code></pre></div> <form class=\\"flex flex-col min-w-max\\" id=\\"s1\\"><div class=\\"flex flex-col s-c1k-_LTyZdv1\\"><label class=\\"label s-c1k-_LTyZdv1\\" for=\\"root\\">Integer min max</label> <input id=\\"root\\" type=\\"number\\" step=\\"1\\" min=\\"0\\" max=\\"10\\">  </div><!--<Layout>--><!--<IntegerField>--></form><!--<Form>--></div><!--<ExampleLayout>--><div class=\\"columns-2 m-4\\"><div class=\\"rounded box-border border-amber border border-rounded bg-gray-3 py-2 px-4\\"><pre class=\\"inline-block \\"><code>{
  \\"type\\": \\"object\\",
  \\"title\\": \\"Integer\\",
  \\"properties\\": {
    \\"integer\\": {
      \\"type\\": \\"integer\\",
      \\"title\\": \\"Integer\\",
      \\"default\\": 0
    }
  }
}</code></pre></div> <form class=\\"flex flex-col min-w-max\\" id=\\"s2\\"></form><!--<Form>--></div><!--<ExampleLayout>--><div class=\\"columns-2 m-4\\"><div class=\\"rounded box-border border-amber border border-rounded bg-gray-3 py-2 px-4\\"><pre class=\\"inline-block \\"><code>{
  \\"type\\": \\"object\\",
  \\"title\\": \\"Integer\\",
  \\"properties\\": {
    \\"integer\\": {
      \\"type\\": \\"integer\\",
      \\"title\\": \\"Integer\\",
      \\"default\\": 1,
      \\"minimum\\": 0,
      \\"maximum\\": 10
    }
  }
}</code></pre></div> <form class=\\"flex flex-col min-w-max\\" id=\\"s3\\"></form><!--<Form>--></div><!--<ExampleLayout>--><!--<Integer>--></div>"
`;
